app/importers/iris_mapper.rb

```diff
@@ -16,8 +16,24 @@ def shapefiles
     metadata['shape_file']
   end
 
+  def identifier
```

Comment:
It looks like geoblacklight uses the dc:identifier field for the record's fedora ID, so we shouldn't have it in the CSV file.

Reply:
@val99erie I took it out.